@xen-orchestra/backups/_incrementalVm.mjs

```diff
@@ -12,24 +11,18 @@ import { cancelableMap } from './_cancelableMap.mjs'
 import { Task } from './Task.mjs'
 import pick from 'lodash/pick.js'
 
+// in `other_config` of an incrementally replicated VM, contains the UUID of the source VM
 export const TAG_BASE_DELTA = 'xo:base_delta'
```

Comment:
This constant is not used in https://github.com/vatesfr/xen-orchestra/blob/9b635660631895607b6594563407be883f03b4b3/%40xen-orchestra/backups/_runners/_writers/IncrementalRemoteWriter.mjs#L198